src/Files.App/Views/Settings/AdvancedPage.xaml

```diff
@@ -162,8 +162,7 @@
 			<local:SettingsBlockControl
 				x:Name="ReplaceOpenFileDialogSetting"
 				Title="{helpers:ResourceString Name=SettingsSetAsOpenDialog}"
-				HorizontalAlignment="Stretch"
-				x:Load="False">
+				HorizontalAlignment="Stretch">
```

Comment:
```suggestion
				x:Load="False"
				HorizontalAlignment="Stretch">
```

Reply:
Alternatively we can keep it visible in debug mode

Reply:
Done!